srcalc/src/main/java/gov/va/med/srcalc/web/view/DisplayItem.java

```diff
@@ -0,0 +1,65 @@
+package gov.va.med.srcalc.web.view;
+
+import com.github.rjeschke.txtmark.Processor;
+import com.google.common.base.Optional;
+
+import gov.va.med.srcalc.domain.model.VariableGroup;
+
+/**
+ * Contains common properties for items to display to the user.
+ */
+public abstract class DisplayItem
```

Comment:
I think this type hierarchy is pretty good, but I find the technique of the superclass storing properties and subclasses passing them via the constructor a little strange. Let's discuss.